distributions/dependencies.json

```diff
@@ -200,9 +200,7 @@
     "sentencepiece",
     "tqdm",
     "transformers",
-    "uvicorn",
-    "sentence-transformers --no-deps",
```

Comment:
dropping these on purpore?

Reply:
Yes. This is automated by distro_codegen script.